core/src/main/scala/akka/persistence/cassandra/query/EventsByPersistenceIdStage.scala

```diff
@@ -318,20 +331,7 @@ import scala.compat.java8.FutureConverters._
 
       override def preStart(): Unit = {
         queryState = QueryInProgress(switchPartition = false, fetchMore = false, System.nanoTime())
-        session.highestDeletedSequenceNumber(persistenceId).onComplete {
-          getAsyncCallback[Try[Long]] {
-            case Success(delSeqNr) =>
-              // lowest possible seqNr is 1
-              expectedNextSeqNr = math.max(delSeqNr + 1, math.max(fromSeqNr, 1))
-              partition = partitionNr(expectedNextSeqNr)
-              // initial query
-              queryState = QueryIdle
-              query(switchPartition = false)
-
-            case Failure(e) => onFailure(e)
-
-          }.invoke
-        }
+        session.highestDeletedSequenceNumber(persistenceId).onComplete(highestDeletedSequenceNrCb.invoke)
 
```

Comment:
there is one more in `EventsByPersistenceIdStage.checkForGaps`

Reply:
My grep skills are weak! Doh!